AirCasting/SessionViews/ReconnectSessionCardView.swift

```diff
@@ -0,0 +1,95 @@
+// Created by Lunar on 21/10/2022.
+//
+
+import SwiftUI
+import AirCastingStyling
+import Resolver
+
+struct ReconnectSessionCardView: View {
+    let session: SessionEntity
```

Comment:
do we really need this here? Maybe we can use `Sessionable` instead? We're trying not to use CoreData entities in views anymore